src/AddTodoForm.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+const AddTodoForm = () => {
+  return (
+    <form>
+      <label htmlFor="todoTitle">Title</label>
+      <input type="text" id="todoTitle" />
+      <input type="submit" value="Add" />
```

Comment:
Good use of an input type="submit"! It results in the same type of button but is more semantically correct. Bear in mind that you may have to make minor adaptations to the instructions in future lessons as they will work with a button element. I applaud this choice but if you run into any issues, please reach out on Slack.

Reply:
Thank you